javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,67 @@
+// Movies exercise -------------------------------------------------
+const getDataFromApi = async () => {
+  const apiUrl =
+    'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json';
+
+  const requestData = await fetch(apiUrl);
+  const data = await requestData.json();
+
+  const badMoviesArray = data.filter((movie) => movie.rating < 5);
```

Comment:
instead of filtering the movies array 2 times i would suggest to write the condition in first filter.